src/commands/deployAzurePipeline/configurePipeline.ts

```diff
@@ -0,0 +1,50 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from 'vscode-azureextensionui';
+
+export async function configurePipeline(context: IActionContext, target: any): Promise<void | undefined> {
+    const deployToAzureExtensionInstalled = await isDeployToAzureExtensionInstalled();
+    if (!deployToAzureExtensionInstalled) {
+        return undefined;
+    }
+
+    const cloudExplorer = await k8s.extension.cloudExplorer.v1;
+    if (!cloudExplorer.available) {
+        return undefined;
+    }
+
+    const clusterTarget = cloudExplorer.api.resolveCommandTarget(target);
+    if (clusterTarget && clusterTarget.cloudName === "Azure" && clusterTarget.nodeType === "resource" && clusterTarget.cloudResource.nodeType === "cluster") {
+        const configurePipelineCommand = 'configure-cicd-pipeline';
+        await executeDeployToAzureExtensionInstalled(configurePipelineCommand, clusterTarget);
+    } else {
+        vscode.window.showInformationMessage('This command only applies to AKS clusters.');
+        return undefined;
+    }
+
+}
+
+async function isDeployToAzureExtensionInstalled(): Promise<boolean> {
+    const deployToAzureExtensionId = 'ms-vscode-deploy-azure.azure-deploy';
```

Comment:
Consider pulling this out to a top-level constant (in `SHOUTY_CAPS`) so it jumps out as a true 'magic string' constant rather than just an immutable variable.

Reply:
updated